src/server/utilities/canAdvertise/index.js

```diff
@@ -0,0 +1,16 @@
+const canAdvertise = headers => {
+  const isAdvertiseCombine = headers['x-ip_is_advertise_combined'];
+
+  if (isAdvertiseCombine && isAdvertiseCombine === 'yes') {
+    return true;
+  }
+
+  const isUK = headers['x-bbc-edge-isuk'];
```

Comment:
Hey! Sorry, I'm still not sure about this logic. What I meant to say with my previous comment is that if `'x-ip_is_advertise_combined'` exists, we should not check `'x-bbc-edge-isuk'` at all, no matter its value. For example, if `'x-ip_is_advertise_combined'` is `'no'`, we should respect that rather than also checking the value of `'x-bbc-edge-isuk'`, as this header is less accurate. Sorry if that was unclear.

Reply:
This situation could also be covered by a unit test to make it clearer!

Reply:
Alright, so then we can go for what you suggested before https://github.com/bbc/simorgh/pull/7081#discussion_r449553555, which will return either true or false as long as `ip_is_advertise_combined` is defined.

Reply:
Yes, it was a little unclear initially. But I get it now. I'll add the appropriate unit tests as well

Reply:
This is has been updated and also added a comment to explain this. @tochwill